diff --git a/cfgui/upmpdcli-config.pro.in b/cfgui/upmpdcli-config.pro.in
index ff7b485..f8b4d66 100644
--- a/cfgui/upmpdcli-config.pro.in
+++ b/cfgui/upmpdcli-config.pro.in
@@ -2,7 +2,7 @@ TEMPLATE	= app
 LANGUAGE	= C++
 
 VPATH = @srcdir@
-QMAKE_CXXFLAGS += -std=c++11 -I../ -DENABLE_XMLCONF
+QMAKE_CXXFLAGS += -std=c++14 -I../ -DENABLE_XMLCONF
 	       
 
 DEFINES += MDU_INCLUDE_LOG=\'<libupnpp/log.h>\'
