diff --git a/Makefile.am b/Makefile.am
index 08f5f58..4481622 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -264,6 +264,7 @@ install-data-hook: web.tar.gz rdpl2stream.tar.gz uprclbottle.tar.gz
 	(cd $(DESTDIR)/$(cdpluginsspotifydir); chmod a+x spotify-app.py) 
 	(cd $(DESTDIR)/$(cdpluginsuprcldir); chmod a+x uprcl-app.py) 
 	(cd $(DESTDIR)/$(radioscriptsdir);chmod a+rx . *)
+	(cd $(DESTDIR)/$(radioscriptsdir);chmod a-x radiolist.conf)
         
 dist_pkgdata_SCRIPTS = samplescripts/Analog-Input
 
diff --git a/samplescripts/Analog-Input b/samplescripts/Analog-Input
old mode 100644
new mode 100755
index ac956fd..87cff68
--- a/samplescripts/Analog-Input
+++ b/samplescripts/Analog-Input
@@ -1,4 +1,4 @@
-#!/usr/bin//python
+#!/usr/bin/python3
 
 from __future__ import print_function
 
@@ -190,4 +190,3 @@ while True:
 
 time.sleep(1)
 cleanup(0)
-
